### ISSUE TITLE
Reconnect button reconnects to wrong server

### ISSUE THREAD

**Issue by CRD716** [MeteorDevelopment/meteor-client#3788]
### Describe the bug

When I lose connection to a server and press the Reconnect button, it connects me to the first server in my server list instead of the server I was on.

### Steps to reproduce

```bash
Have multiple servers on your server list
Connect to server on latest dev build
Lose connection through some means (unplug your ethernet if you want)
Attempt to reconnect
Watch as you connect to a server that might not like your killaura kicking in
```

### Meteor Version

0.5.4 Dev

### Minecraft Version

1.20.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by ManInMyVan** [comment]
Yeah it's kinda annoying.

**Comment by RedCarlos26** [comment]
I can confirm this

**Comment by Fiz-Victor** [comment]
I created a [pr ](https://github.com/MeteorDevelopment/meteor-client/pull/3438/files#diff-c99db43bdb906d4edcc7fe113d7f88a2ef9ea934f3b010fc249c839c9719ae23)  to fix this but this was closed

**Comment by Wide-Cat** [comment]
should be fixed as of 3e3f2964da06addf50f1b654041a78e7182a8110
